lib/SIL/SILInstructions.cpp

```diff
@@ -2484,6 +2486,32 @@ DestructureTupleInst *DestructureTupleInst::create(SILModule &M,
 }
 
 // SWIFT_ENABLE_TENSORFLOW
+GraphOperationInst::GraphOperationInst(
+    SILModule &M, SILDebugLocation loc, Identifier name,
+    ArrayRef<SILValue> arguments, ArrayRef<GraphOperationAttribute> attrs,
+    ArrayRef<SILType> resultTypes,
+    ArrayRef<ValueOwnershipKind> resultOwnerships) :
+  InstructionBase(loc),
+  MultipleValueInstructionTrailingObjects(this, resultTypes,
+                                          resultOwnerships),
+  Name(name), NumOperands(arguments.size())
+{
+  auto allOperands = getAllOperands();
+  for (unsigned i : indices(arguments))
+    new (&allOperands[i]) Operand(this, arguments[i]);
+  auto attrBuf = AlignedAlloc(sizeof(GraphOperationAttribute) * attrs.size(),
+                              alignof(GraphOperationAttribute));
+  Attributes = MutableArrayRef<GraphOperationAttribute>(
+    static_cast<GraphOperationAttribute *>(attrBuf), attrs.size());
+  std::uninitialized_copy(attrs.begin(), attrs.end(), Attributes.data());
+}
+
+GraphOperationInst::~GraphOperationInst() {
+  for (auto &operand : getAllOperands())
+    operand.~Operand();
+  delete getAttributes().data();
```

Comment:
This should use:
 delete[] getAttributes.data();

Reply:
Done, thanks!